cpp/src/hash/concurrent_unordered_map.cuh

```diff
@@ -17,37 +17,34 @@
 #ifndef CONCURRENT_UNORDERED_MAP_CUH
 #define CONCURRENT_UNORDERED_MAP_CUH
 
-#include <iterator>
-#include <type_traits>
-#include <cassert>
-#include <iostream>
-#include <thrust/pair.h>
-
-#include "cudf.h"
-#include "groupby/aggregation_operations.hpp"
-
+#include <cudf.h>
```

Comment:
Do all the cuDF includes here necessary? I'd like to see if we can avoid dependencies on cuDF as much as possible, since this class may be used outside cuDF for other applications.

Reply:
You're right, these were unnecessary.